docs/pages/kubernetes-access/introduction.mdx

```diff
@@ -1,13 +1,12 @@
 ---
 title: Kubernetes Access
-description: Teleport Kubernetes Access introduction, demo and resources
+description: Teleport Kubernetes Access introduction, demo, and resources.
 ---
 
-Teleport provides a unified access for Kubernetes clusters.
+Teleport provides unified access for Kubernetes clusters.
```

Comment:
Should we drop unified?

Reply:
Actually, yes. Yes, we should. Thanks for catching that.